platform/harmony/api.go

```diff
@@ -13,6 +13,8 @@ type Platform struct {
 	client Client
 }
 
+const Annual = 10
```

Comment:
we need to be dynamic to avoid changing

Reply:
changed it to rely on rpc call which will be implemented.